src/WebJobs.Script/Description/Binding/DataType.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    /// <summary>
+    /// Enumeration of binding value data types.
+    /// </summary>
+    public enum DataType
+    {
+        String,
+        Binary,
+        Stream
```

Comment:
Stream is only intended to be used internally currently. However, we can investigate adding Streaming support to Node (what you really want for image processing), but that will be a bit of work (we'll need to define an interop between .NET Stream and Node - Edge.js doesn't support Streams currently).